src/schema/index.ts

```diff
@@ -18,17 +18,21 @@ const Schema = gql`
     password: String
   }
   type Profile {
-    id: ID!
+    _id: ID!
```

Comment:
Also, here  I would argue that the `schema`should retain the `id` field not the `_id` which is in reference to mongoose style of having the prefix on the id.

Though this would be a matter of preference, so feel free to resolve it.